lib/utils/logging.go

```diff
@@ -20,23 +20,70 @@ import (
 	"io/ioutil"
 	"log/syslog"
 	"os"
+	"strconv"
+	"strings"
 
 	"github.com/gravitational/gravity/lib/defaults"
 
 	log "github.com/sirupsen/logrus"
 	syslogrus "github.com/sirupsen/logrus/hooks/syslog"
+	"google.golang.org/grpc/grpclog"
 )
 
 // InitLogging initalizes logging to log both to syslog and to a file
-func InitLogging(logFile string) {
+func InitLogging(level log.Level, logFile string) {
 	log.StandardLogger().Hooks.Add(&Hook{
 		path: logFile,
 	})
-	setLoggingOptions()
+	setLoggingOptions(level)
 }
 
-func setLoggingOptions() {
-	log.SetLevel(log.DebugLevel)
+// InitGRPCLoggerFromEnvironment configures the GRPC logger if any of the related environment variables
+// are set.
+func InitGRPCLoggerFromEnvironment() {
+	const (
+		envSeverityLevel  = "GRPC_GO_LOG_SEVERITY_LEVEL"
```

Comment:
Can you please add information about using these environment variables to our docs, e.g. troubleshooting section. I never remember what these variables are and how to use them, let's document it once and for all.

Reply:
Ok, will do. On a side note, given that these are sort of implementation detail, this changeset will  automatically turn on gRPC logging if `--debug` has been specified. Now, this might be something to take with a grain of salt since these logs are output directly to stdout - so if we decide it is not worth it, I can remove that bit so the logs can only be controlled via environment.